locales/en.json

```diff
@@ -1,180 +1,188 @@
-  "Type is required": "Type is required",
-  "Retrieved success": "Retrieved success",
-  "Success updated": "Success updated",
-  "Please fill your location": "Please fill your location",
-  "Please fill your destination": "Please fill your destination",
-  "Please enter a valid date like YYY-MM-DD": "Please enter a valid date like YYY-MM-DD",
-  "Please fill your travel reason": "Please fill your travel reason",
-  "Please choose accomodation": "Please choose accomodation",
-  "Please choose type of trip": "Please choose type of trip",
-  "Travel reason is required": "Travel reason is required",
-  "Name should be string": "Name should be string",
-  "Name input can't be empty": "Name input can't be empty",
-  "Name property is required": "Name property is required",
-  "Location should be string": "Location should be string",
-  "Location input can't be empty": "Location input can't be empty",
-  "Location should be provided": "Location should be provided",
-  "Description should be of type string": "Description should be of type string",
-  "Please, provide meaningful description": "Please, provide meaningful description",
-  "At least one room is required": "At least one room is required",
-  "Please provide a valid url": "Please provide a valid url",
-  "Provide image for your accomodation": "Provide image for your accomodation",
-  "Image is required": "Image is required",
-  "Accomodation created successfully": "Accomodation created successfully",
-  "Room in facility name created successfuly": "Room in facility name created successfuly",
-  "Accomodation successfully updated": "Accomodation successfully updated",
-  "Room Deleted successfully": "Room Deleted successfully",
-  "Accomodation deleted successfully": "Accomodation deleted successfully",
-  "Reset password - Barefoot Nomad": "Reset password - Barefoot Nomad",
-  "Failed in sending Reset password email !!!": "Failed in sending Reset password email !!!",
-  "A link to reset your password has been sent to your email address !!!": "A link to reset your password has been sent to your email address !!!",
-  "Email does NOT exist in our database !!!": "Email does NOT exist in our database !!!",
-  "The passwords do NOT match !!!": "The passwords do NOT match !!!",
-  "Password well reset": "Password well reset",
-  "Please fill in the confirmation": "Please fill in the confirmation",
-  "Confirmation is required": "Confirmation is required",
-  "Invalid token": "Invalid token",
-  "jwt expired": "jwt expired",
-  "Accomodation not found": "Accomodation not found",
-  "No trips found": "No trips found",
-  "Not found": "Not found"
-}
+	"Welcome to barefoot nomad": "Welcome to barefoot nomad",
+	"Email already exists": "Email already exists",
+	"Successfully registered": "Successfully registered",
+	"Email or password is incorrect": "Email or password is incorrect",
+	"Your are successfully loged in": "Your are successfully loged in",
```

Comment:
Look out for these spacing changes, and add an empty line at the end of this file.